order/src/main/java/com/microservicestudy/order/domain/mapper/OrderMapper.java

```diff
@@ -4,29 +4,30 @@
 import com.microservicestudy.order.domain.entity.OrderItem;
 import com.microservicestudy.order.domain.request.OrderItemRequest;
 import com.microservicestudy.order.domain.request.OrderRequest;
-import com.microservicestudy.order.response.OrderItemResponse;
-import com.microservicestudy.order.response.OrderResponse;
+import com.microservicestudy.order.domain.response.OrderItemResponse;
+import com.microservicestudy.order.domain.response.OrderResponse;
+import com.microservicestudy.order.domain.response.StoreResponse;
 
 import java.util.ArrayList;
 
 public class OrderMapper {
 
-    public static OrderResponse toResponse(Order order) {
+    public static OrderResponse toResponse(Order order, StoreResponse storeResponse) {
         OrderResponse orderResponse = new OrderResponse();
 
-        orderResponse.setStore(order.getStore());
+        orderResponse.setId(order.getId());
+        orderResponse.setStore(storeResponse);
         orderResponse.setAddress(order.getAddress());
         orderResponse.setItems(new ArrayList<>());
-        for (OrderItem item : order.getItems()) {
 
+        for (OrderItem item : order.getItems()) {
```

Comment:
Hmm... Agora que já estamos mais acostumados com o projeto, será que conseguimos utilizar alguma feature do Java8 para melhorar esse for each?